core/src/main/java/com/linecorp/armeria/server/management/ThreadDumpService.java

```diff
@@ -49,4 +52,9 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exc
             return HttpResponse.of(HttpStatus.OK, MediaType.PLAIN_TEXT, threadDump);
         }
     }
+
+    @Override
+    public ExchangeType exchangeType(RequestHeaders headers, Route route) {
+        return ExchangeType.UNARY;
```

Comment:
We should only do this when `:method` is `GET`.

Reply:
As the response is a fixed stream message, I thought, we could use `UNARY` for any `:method`.
Could you elaborate on why we use `UNARY` only for `GET`?

Reply:
Ah, you meant that we don't need to aggregate a request for this `HttpService`.
That makes sense.

Reply:
At second thought, it's probably OK to always return `UNARY` here because:

- A `GET` request can have a body anyway
- This service is for internal use only; and
- Server already enforces a limit on the request body length.

Could you revert to the original behavior? :pray:

Reply:
Sound good! Let me revert this.